unime/src-tauri/tests/tests/search_query.rs

```diff
@@ -84,3 +120,11 @@ async fn test_credential_delete_recent() {
     )
     .await;
 }
+
+#[test]
+fn tets() {
```

Comment:
Should this "tets" really be part of the PR? ;)

Reply:
nope haha